pacman/operations/router_algorithms/ner_route.py

```diff
@@ -83,7 +84,7 @@ def _convert_a_route(
             routing_tables, partition, None, next_incoming_link, next_hop)
```

Comment:
cant have None there. the docs for _convert_a_route on lines 54 and 55 say it must be a int.

Reply:
If you want to do a tidy up of existing code, please feel free.

Reply:
..................... pooy to you then. lol.